test/build/50_fakeroot.bats

```diff
@@ -693,3 +709,191 @@ EOF
+RUN true
+EOF
+    echo "$output"
+    [[ $status -eq 0 ]]
+    [[ $output = *'available --force: fedora'* ]]
+}
+
+@test "${tag}: Fedora 26: unneeded, no --force, build fails" {
+    scope full
+    # no commands that may need it, without --force, build fails
+    run ch-image -v build -t fakeroot-temp -f - . <<'EOF'
+FROM fedora:26
+RUN false
+EOF
+    echo "$output"
+    [[ $status -eq 1 ]]
+    [[ $output = *"build failed: current version of --force wouldn't help"* ]]
+    [[ $output = *'build failed: RUN command exited with 1'* ]]
+}
+
+@test "${tag}: Fedora 26: unneeded, with --force" {
+    scope full
+    # no commands that may need it, with --force, warning
+    run ch-image -v build --force -t fakeroot-temp -f - . <<'EOF'
+FROM fedora:26
+RUN true
+EOF
+    echo "$output"
+    [[ $status -eq 0 ]]
+    [[ $output = *'warning: --force specified, but nothing to do'* ]]
+}
+
+@test "${tag}: Fedora 26: maybe needed but actually not, no --force" {
+    scope full
+    # commands that may need it, but turns out they don’t, without --force
+    run ch-image -v build -t fakeroot-temp -f - . <<'EOF'
+FROM fedora:26
+RUN dnf install -y ed
+EOF
+    echo "$output"
+    [[ $status -eq 0 ]]
+    [[ $output = *'available --force'* ]]
+    [[ $output = *'RUN: available here with --force'* ]]
+}
+
+@test "${tag}: Fedora 26: maybe needed but actually not, with --force" {
+    scope full
+    # commands that may need it, but turns out they don’t, with --force
+    run ch-image -v build --force -t fakeroot-temp -f - . <<'EOF'
+FROM fedora:26
+RUN dnf install -y ed
+EOF
+    echo "$output"
+    [[ $status -eq 0 ]]
+    [[ $output = *'will use --force'* ]]
+    [[ $output = *'--force: init OK & modified 1 RUN instructions'* ]]
+}
+
+@test "${tag}: Fedora 26: needed but no --force" {
+    scope full
+    # commands that may need it, they do, fail & suggest
+    run ch-image -v build -t fakeroot-temp -f - . <<'EOF'
+FROM fedora:26
+RUN dnf install -y --setopt=install_weak_deps=false openssh
+EOF
+    echo "$output"
+    [[ $status -eq 1 ]]
+    [[ $output = *'available --force'* ]]
+    [[ $output = *'RUN: available here with --force'* ]]
+    [[ $output = *'build failed: --force may fix it'* ]]
+    [[ $output = *'build failed: RUN command exited with 1'* ]]
+}
+
+@test "${tag}: Fedora 26: needed, with --force" {
+    scope standard
+    # commands that may need it, they do, --force, success
+    run ch-image -v build --force -t fakeroot-temp -f - . <<'EOF'
+FROM fedora:26
+RUN dnf install -y --setopt=install_weak_deps=false openssh
+EOF
+    echo "$output"
+    [[ $status -eq 0 ]]
+    [[ $output = *'will use --force'* ]]
+    [[ $output = *'--force: init OK & modified 1 RUN instructions'* ]]
+}
+
+@test "${tag}: Fedora 35: unneeded, no --force, build succeeds" {
+    scope standard
+    # no commands that may need it, without --force, build succeeds
+    # also: correct config
+    run ch-image -v build -t fakeroot-temp -f - . <<'EOF'
+FROM fedora:35
```

Comment:
Similar to above, can we do `fedora:latest` here? (which seems to be Fedora 33 currently) Goal being to test the latest released Fedora, not a pre-release, though? Or maybe we really do want `fedora:rawhide` so we can fix problems early?

Reply:
Let's do `fedora:latest`, I know nothing about how stable their pre-release images are.